### PR TITLE
Rename Mac architecture on Quick Start page

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Feature request by @traceon 

Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by traceon** [comment]
macOS

File: website/templates/index/quickstart.html [hunk 1]
```diff
@@ -20,10 +20,10 @@ <h2 id="quick-start" class="display-2 mb-5 mb-md-8 mb-lg-10 text-center">Quick S
                     <a class="nav-link" id="arm-tab" data-toggle="tab" href="#arm" role="tab" aria-controls="arm" aria-selected="false" title="ARM packages">Linux (ARM)</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">Mac (x86)</a>
+                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">Mac (Intel)</a>
```
Comment (traceon) [inline website/templates/index/quickstart.html:23]: macOS

Comment (traceon) [inline website/templates/index/quickstart.html:23]:
```suggestion
                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">macOS (Intel)</a>
```

File: website/templates/index/quickstart.html [hunk 2]
```diff
@@ -20,10 +20,10 @@ <h2 id="quick-start" class="display-2 mb-5 mb-md-8 mb-lg-10 text-center">Quick S
...
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" id="mac-arm-tab" data-toggle="tab" href="#mac-arm" role="tab" aria-controls="mac-arm" aria-selected="false" title="Mac ARM packages">Mac (ARM)</a>
+                    <a class="nav-link" id="mac-arm-tab" data-toggle="tab" href="#mac-arm" role="tab" aria-controls="mac-arm" aria-selected="false" title="Mac ARM packages">Mac (Apple Silicon)</a>
```
Comment (traceon) [inline website/templates/index/quickstart.html:26]:
```suggestion
                    <a class="nav-link" id="mac-arm-tab" data-toggle="tab" href="#mac-arm" role="tab" aria-controls="mac-arm" aria-selected="false" title="Mac ARM packages">macOS (Apple silicon)</a>
```

File: website/templates/index/quickstart.html [hunk 3]
```diff
@@ -20,10 +20,10 @@ <h2 id="quick-start" class="display-2 mb-5 mb-md-8 mb-lg-10 text-center">Quick S
...
                 </li>
                 <li class="nav-item">
                     <a class="nav-link" id="freebsd-tab" data-toggle="tab" href="#freebsd" role="tab" aria-controls="freebsd" aria-selected="false" title="FreeBSD packages">FreeBSD</a>
```
Comment (traceon) [inline website/templates/index/quickstart.html:29]:
```suggestion
                    <a class="nav-link" id="freebsd-tab" data-toggle="tab" href="#freebsd" role="tab" aria-controls="freebsd" aria-selected="false" title="FreeBSD packages">FreeBSD (x86_64)</a>
```

File: website/templates/index/quickstart.html (hunk 2 above)
Comment (alexey-milovidov) [reply website/templates/index/quickstart.html:26]: Why "silicon" in lower case?

File: website/templates/index/quickstart.html (hunk 1 above)
Comment (alexey-milovidov) [reply website/templates/index/quickstart.html:23]: Why `macOS` instead of `MacOS`?

Comment (traceon) [reply website/templates/index/quickstart.html]: https://fortune.com/2016/06/13/apple-mac-os-sierra/

Comment (traceon) [reply website/templates/index/quickstart.html]: https://www.apple.com/macos/monterey/

File: website/templates/index/quickstart.html (hunk 2 above)
Comment (traceon) [reply website/templates/index/quickstart.html:26]: That's how it is called officially:  https://support.apple.com/en-us/HT211814

**Pull request merged**